cmd/checkpoint/main.go

```diff
@@ -31,25 +31,15 @@ const (
 	kubeAPIServer = "kube-apiserver"
 )
 
+var podAPIServerMeta = unversioned.TypeMeta{
```

Comment:
s/podAPIServerMeta/podMeta/ -- not specific to apiServer, and we'll need to use it for other checkpointed pods.